packages/flutter_tools/test/commands.shard/permeable/build_aar_test.dart

```diff
@@ -259,3 +313,15 @@ class FakeAndroidBuilder extends Fake implements AndroidBuilder {
     this.buildNumber = buildNumber;
   }
 }
+
+class FakeAndroidSdk extends Fake implements AndroidSdk {
+  FakeAndroidSdk(this.directory);
+
+  @override
+  final Directory directory;
+}
```

Comment:
Overriding `directory` isn't necessary.
```suggestion
class FakeAndroidSdk extends Fake implements AndroidSdk { }
```

Reply:
done